### PR TITLE
Fix partially deposited status bug

### LINKED ISSUE #7137: awaiting activation messages are off
message: 
[2020-08-28 10:53:38]  INFO validator: Validator status activationEpoch=18446744073709551615 index=50289 pubKey=0x86984862c1c7 status=PARTIALLY_DEPOSITED

Off:
1. status=PARTIALLY_DEPOSITED
should be DEPOSITED, cause it is full 32 eth

2. activationEpoch=18446744073709551615
that is an infinite number, not the actual epoch

### PR DESCRIPTION
**Pull request by michaelhly**
**What type of PR is this?**
Bug fix

**What does this PR do? Why is it needed?**
`DEPOSITED` statuses are getting overridden to `PARTIALLY_DEPOSITED`.

**Which issues(s) does this PR fix?**

Fixes #7137

**Other notes for review**

### CODE REVIEW

File: beacon-chain/rpc/validator/status.go [hunk 1]
```diff
@@ -202,10 +203,14 @@ func (vs *Server) validatorStatus(
 			} else {
 				// Check if there was a deposit deposit.
 				deposit, eth1BlockNumBigInt := vs.DepositFetcher.DepositByPubkey(ctx, pubKey)
-				if eth1BlockNumBigInt != nil && deposit.Data.Amount >= params.BeaconConfig().MaxEffectiveBalance {
-					resp.Status = ethpb.ValidatorStatus_DEPOSITED
-				} else if eth1BlockNumBigInt != nil && deposit.Data.Amount > 0 {
-					resp.Status = ethpb.ValidatorStatus_PARTIALLY_DEPOSITED
+				if eth1BlockNumBigInt != nil {
```
Comment (terencechain) [inline beacon-chain/rpc/validator/status.go:206]: Want to add a test for this logic?

Comment (michaelhly) [reply beacon-chain/rpc/validator/status.go:206]: 😅 Okay. I'll have to investigate how to actually get these to trigger.

Comment (terencechain) [reply beacon-chain/rpc/validator/status.go:206]: Can always refactor this out into a helper function for testability :)

**Comment by terencechain** [comment]
Should be fixed by #7138

**Comment by michaelhly** [comment]
@terencechain Added tests.

**Pull request merged**